app/views/govuk_publishing_components/components/_label.html.erb

```diff
@@ -24,16 +24,16 @@
     <% end %>
   >
     <%= text %>
+    <% if hint_text.present? %>
```

Comment:
I did a bunch of work to move the hint outside the label, tested it extensively.

But if it's easier might be better to have parity with GOV.UK Frontend, even if it's somewhat a regression.

If we do move it back in, we should remove the code and tests for this specific behaviour.

Reply:
https://github.com/alphagov/govuk_elements/issues/574

https://github.com/alphagov/government-frontend/commit/786ab1df467373f4b2e26e83102a52e26fd03218

Reply:
Thanks Nick. As you were the author I figured it was intentional but couldn't see anything in the commit history and then referred to GOV.UK Frontend as canonical source. I'll back that change out and refactor the fixes, but also make sure the accessibility context is in the docs :+1:

Reply:
The commit history was somewhat lost by moving the work into this repository, sorry.